prow/plugins/assign/assign.go

```diff
@@ -27,23 +27,37 @@ import (
 	"k8s.io/test-infra/prow/plugins"
 )
 
-const pluginName = "assign"
+const (
+	pluginNameAssign = "assign"
+	pluginNameReview = "review"
```

Comment:
It should either be in the same plugin as the `/assign` code or in a separate package.